sql/updates/mangos/areatriggerteleport_condition.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE areatrigger_teleport ADD COLUMN `condition_id` INT(11) unsigned NOT NULL default '0' AFTER status_failed_text;
+
```

Comment:
that pretty much does not matter, I just added it at the end of the table as customary in small tables